py3status/modules/spaceapi.py

```diff
@@ -3,26 +3,27 @@
 Display status of a given hackerspace.
 
 Configuration parameters:
-    button_url: Button that when clicked opens the URL sent in the space's API.
-        Setting to None disables. (default 3)
-    cache_timeout: Set timeout between calls in seconds (default 60)
-    closed_text: text if space is closed, strftime parameters
-        will be translated (default 'closed')
-    open_text: text if space is open, strftime parmeters will be translated
-        (default 'open')
-    time_text: format used for time display (default ' since %H:%M')
-    url: URL to SpaceAPI json file of your space
+    button_url: mouse button to open URL sent in space's API (default 3)
+    cache_timeout: refresh interval for this module (default 60)
+    format: display format for this module (default '{spaceapi}')
+    icon_closed: show when hackerspace is closed (default 'closed')
+    icon_open: show when hackerspace is open (default 'open')
+    icon_time: display format for time (default ' since %H:%M')
+    url: specify JSON URL of a hackerspace to retrieve from
         (default 'http://status.chaospott.de/status.json')
 
+__Note: Strftime parameters in icon_* parameters will be translated.__
+
 Color options:
-    color_closed: Space is open, defaults to color_bad
-    color_open: Space is closed, defaults to color_good
+    color_bad: Space closed
+    color_open: Space open
```

Comment:
Yes. Thank you for catching that. I think I had a discussion w/ @tobes about good/bad, on/off, up/down, closed/open, etc... Maybe we should explicitly cover all scenarios in `post_config_hook`. It's either that or making people stick with only `good/bad` like here.